server/vulnerabilities/nvd/cpe_translations.json

```diff
@@ -360,6 +360,15 @@
       "vendor": ["microsoft"]
     }
   },
+  {
+    "software": {
+      "name": ["/golang\\.go/"],
+      "source": ["vscode_extensions"]
+    },
+    "filter": {
+      "skip": true
```

Comment:
Running  `SELECT * FROM cpe_2 WHERE target_sw LIKE '%visual_studio_code%' AND product LIKE '%go%’;` on the CPE sqlite db from NVD returned no rows containing "golang.go", or even “go” at all, indicating there are _no known (as of 10/3/25) CVEs in this extension_, meaning it’s safe to skip matching this SW name to avoid the very broad match glob, `cpe:2.3:a:golang:go:*:*:*:*:*:*:*:*`, provided by NVD